### DISCUSSION TITLE
xmake新增libllvm包，check没问题，编译报错提示找不到头文件，include没有包含进去

### DISCUSSION THREAD

**Discussion by plutolove** [xmake-io/xmake#3674] [Q&A]
新增配置：https://github.com/plutolove/xmake-repo/blob/dev/packages/l/libllvm/xmake.lua
checking for xmake::libllvm ... libllvm 15.0.7
![image](https://user-images.githubusercontent.com/8512477/233840249-99ba2103-291a-4f9f-8f6d-d7605349794b.png)

包安装目录:
![image](https://user-images.githubusercontent.com/8512477/233840410-1d560e3b-c421-44a6-9a31-f56af2bd7602.png)

**Comment by plutolove** [comment]
@waruqi 麻烦帮忙看看

**Reply by plutolove** [reply]
找到原因了，我配置错了

**Comment by waruqi** [comment]
官方仓库 llvm 包原本就支持 库集成。。https://github.com/xmake-io/xmake-repo/issues/1802

**Reply by plutolove** [reply]
这个编译太多东西了，我想搞一个只依赖llvm和mlir的库

**Reply by waruqi** [reply]
`add_requires("llvm", {configs = {mlir = true}})` 就是只编译 mlir 。。你可以按需配置，仅仅开启自己需要的模块编译。
